spec/rails_helper.rb

```diff
@@ -1,6 +1,8 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 require 'spec_helper'
 ENV['RAILS_ENV'] ||= 'test'
+ENV['LEGACY_HOST'] ||= "https://test.data.gov.uk"
+ENV['LEGACY_API_KEY'] ||= "123"
```

Comment:
is this still necessary if we use a `.env.test` file with the dotenv gem?

Reply:
CI doesn't have dotenv

Reply:
:(

Reply:
Add it if you want to -- or check, because it might, but it wasn't built in mind with supporting dotenv

Reply:
might add support for it in another PR